apps/src/tutorialExplorer/tutorialDetail.jsx

```diff
@@ -46,9 +49,19 @@ export default class TutorialDetail extends React.Component {
   };
 
   startTutorialClicked = () => {
+    // Send event to Google Analytics
     const shortCode = this.props.item.short_code;
     ga('send', 'event', 'learn', 'start', shortCode);
     ga('send', 'event', 'learn', `start-${this.props.grade}`, shortCode);
+    // Send event to Statsig
+    const activityUrl = this.props.item.url;
+    analyticsReporter.sendEvent(
+      EVENTS.HOC_ACTIVITY_START_BUTTON_CLICKED,
+      {
+        activityUrl: activityUrl,
```

Comment:
@dmcavoy I added the activity url here, should I add anything else like the title?

Reply:
I think thats good to start with. Can't think of others that we won't get from statsig automatically